server_test.go

```diff
@@ -490,32 +489,48 @@ func TestServer_SetDefaultRetentionPolicy_ErrRetentionPolicyNotFound(t *testing.
 
 // Ensure the database can write data to the database.
 func TestServer_WriteSeries(t *testing.T) {
-	s := OpenServer(NewMessagingClient())
+	c := NewMessagingClient()
+	s := OpenServer(c)
 	defer s.Close()
 	s.CreateDatabase("foo")
 	s.CreateRetentionPolicy("foo", &influxdb.RetentionPolicy{Name: "myspace", Duration: 1 * time.Hour})
 	s.CreateUser("susy", "pass", false)
 
+	// Check if a topic is being subscribed to.
+	var subscribed bool
+	c.SubscribeFunc = func(replicaID, topicID uint64) error {
+		subscribed = true
+		return nil
+	}
+
 	// Write series with one point to the database.
-	timestamp := mustParseTime("2000-01-01T00:00:00Z")
-	name := "cpu_load"
 	tags := map[string]string{"host": "servera.influx.com", "region": "uswest"}
 	values := map[string]interface{}{"value": 23.2}
+	if err := s.WriteSeries("foo", "myspace", "cpu_load", tags, mustParseTime("2000-01-01T00:00:00Z"), values); err != nil {
```

Comment:
Might be better to use test strings with the word "retention" rather than "space".